requirements.txt

```diff
@@ -1,9 +0,0 @@
-torchvision>=0.5.0
-pretrainedmodels==0.7.4
-efficientnet-pytorch==0.7.1
-timm==0.9.7
-huggingface_hub>=0.24.6
```

Comment:
This is actually the minimum version (or maybe 0.24.5), older versions do not support arguments for model cards, it was introduced recently.

Reply:
Do we have tests for that yet? Would like to get to 100% test coverage so we know whether or not the tests are actually useful

Reply:
no, we don't. I didn't add a test for hub integration, but definetely it worth adding it :)
It would be great to have 100% coverage, but considering the wide range of backbones and configuration options not sure it is possible.

Reply:
It's definitely possible, it just requires a lot of dedication. TorchGeo has 100% coverage and 4x as many lines of code. For the wide range of backbones, that just looks like a for-loop. If you can add tests for the huggingface_hub code, I can figure out the exact versions required.

Reply:
Ah, I see it now, it just needs https://github.com/huggingface/huggingface_hub/pull/2310